backend/requirements.txt

```diff
@@ -14,8 +14,8 @@ pydantic==1.10.4
 pydantic[email]==1.10.4
 python-dotenv==0.21.0
 python-jose[cryptography]==3.3.0
-redis
-requests
+redis==4.5.5
+requests==2.30.0
```

Comment:
Are these requirement versions needed for the new functionality to work?

Reply:
check the below comment..